### PR TITLE
fix: ascanrules: Source Code Disclosure WEB-INF no longer skipped

### PR DESCRIPTION
**Pull request by kingthorin**
See:
- https://github.com/mstrobel/procyon/issues/12
- zaproxy/zaproxy#4038 / zaproxy/zap-extensions#2338 / 
  - Related to zaproxy/zaproxy#2602
- Reverts zaproxy/zap-extensions#1723

Signed-off-by: kingthorin <kingthorin@users.noreply.github.com>

### CODE REVIEW

File: addOns/ascanrules/src/main/javahelp/org/zaproxy/zap/extension/ascanrules/resources/help/contents/ascanrules.html [hunk 1]
```diff
@@ -146,7 +146,6 @@ <H2>Server Side Include</H2>
 
 <H2>Source Code Disclosure - /WEB-INF</H2>
 Exploit the presence of an unprotected /WEB-INF folder to download and decompile Java classes, to disclose Java source code.<br>
```
Comment (thc202) [inline addOns/ascanrules/src/main/javahelp/org/zaproxy/zap/extension/ascanrules/resources/help/contents/ascanrules.html:148]: The break line can be removed.

**Comment by thc202** [comment]
The resource message needs to be removed.

**Comment by kingthorin** [comment]
Done & done.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**